client/src/App.vue

```diff
@@ -30,6 +30,9 @@ export default {
 
 
 <style>
+body {
+  padding-right: 0px !important; /* Keeps b-modals from messing with the nav bar */
+}
```

Comment:
Whenever the Bootstrap modal was displayed it would add padding to the right of the body. I'm not quite sure why it was doing that and this is hacky, but it takes care of it.